portfolio/src/main/webapp/movies.js

```diff
@@ -1,4 +1,10 @@
-var slideIndex = 1;
+var slideIndex;
```

Comment:
Try not to use `var`. We prefer to use `const`. In this case, you could make it a const by changing the variable assignment to a ternary operator. 

When you do have to create variable that  can't be constant, we prefer to use a `let` declaration. 

ie  `const slideIndex = condition ? value1 : value2`